bundles/com.espressif.idf.launch.serial.core/src/com/espressif/idf/launch/serial/internal/SerialFlashLaunchConfigDelegate.java

```diff
@@ -150,6 +164,34 @@ protected void launchInternal(ILaunchConfiguration configuration, String mode, I
 		DebugPlugin.newProcess(launch, p, String.join(" ", commands)); //$NON-NLS-1$
 	}
 
+	private void flashOverJtag(ILaunchConfiguration configuration, ILaunch launch) throws CoreException {
+		List<String> commands = new ArrayList<>();
+		String openocdExe = configuration.getAttribute(SERVER_EXECUTABLE, DEFAULT_PATH + DEFAULT_EXECUTABLE);
```

Comment:
DEFAULT_EXECUTABLE might be different on windows e.g openocd.exe, please check

Reply:
@kolipakakondal 

Checked on Windows 10, execution of opencocd works